testsuite/W60.py

```diff
@@ -19,6 +19,10 @@
 regex = '''
 \.png$
 '''
+#: W605:2:5
```

Comment:
Would you be willing to add line and position numbers to the other examples of this to prove it works in all cases?

Reply:
done!  looks like this maybe fixes #795 as well:

```python
def func():
    """this is a
    long-ish docstring
    with invalid \d escapes
    """
    pass
```

```console
$ ./pycodestyle.py test.py 
test.py:2:5: W605 invalid escape sequence '\d'
```